cnf-tests/entrypoint/test-run.sh

```diff
@@ -3,19 +3,19 @@ set -e
 # Setting -e is fine as we want both config and validiation to succeed
 # before running the "real" tests.
 
-TEST_SUITES=${TEST_SUITES:-"validationsuite configsuite cnftests"}
+LATENCY_TEST_RUN="${LATENCY_TEST_RUN:true}"
+DISCOVERY_MODE="${DISCOVERY_MODE:true}"
+FEATURES="${FEATURES:performance}"
+
+TEST_SUITES=${TEST_SUITES:-"latency-e2e.test"}
 SUITES_PATH="${SUITES_PATH:-~/usr/bin}"
 
-suites=( $TEST_SUITES )
+suites=($TEST_SUITES)
 if [ "$IMAGE_REGISTRY" != "" ] && [[ "$IMAGE_REGISTRY" != */ ]]; then
     export IMAGE_REGISTRY="$IMAGE_REGISTRY/"
 fi
 
 for suite in "${suites[@]}"; do
```

Comment:
IIUC now we only have the `latency-e2e.test`, so why do we need this loop?

Reply:
Because the intention is for the script to be able to run different test-suites if the user wants to.

Reply:
But this whole split purpose is to support a single suite case, so why not simplified as much as possible?

Reply:
@jlojosnegros sorry for the late reply, I was sick last week.
Tal-or's point is valid, i.e. with this split, the test-run.sh script will be used only for the latency test,
therefore we can simplify it and remove the loop, as it only needs to run the latency-e2e.test.

Reply:
Do not worry at all @liornoy 
Change done. 
Thanks